service-front/module/Application/tests/View/DateCheckViewModelHelperTest.php

```diff
@@ -0,0 +1,324 @@
+            ],
+            'expectedDonorText' => ['You must have signed and dated continuation sheet/s 2 before you signed ' .
+                                    'section 9 of the LPA, or on the same day.'],
+            'expectedAttorneyText' => []
+        ],
+        // LPA has additional information on how replacement attorneys should act (section 4) (generates CS2)
+        [
+            'lpa' => [
+                'document' => [
+                    'replacementAttorneyDecisions' => [
+                        'howDetails' => 'Replacement attorneys should only step in when I say so.'
+                    ]
+                ]
+            ],
+            'expectedDonorText' => ['You must have signed and dated continuation sheet/s 2 before you signed ' .
+                                    'section 9 of the LPA, or on the same day.'],
+            'expectedAttorneyText' => []
+        ],
+        // LPA has additional information on when replacement attorneys should act (generates CS2)
+        [
+            'lpa' => [
+                'document' => [
+                    'replacementAttorneyDecisions' => [
+                        'when' => 'When the primary attorney cannot be contacted'
+                    ]
+                ]
+            ],
+            'expectedDonorText' => ['You must have signed and dated continuation sheet/s 2 before you signed ' .
+                                    'section 9 of the LPA, or on the same day.'],
+            'expectedAttorneyText' => []
+        ],
+        // LPA has additional information in preferences and instructions (section 7) (generates CS2)
+        [
+            'lpa' => [
+                'document' => [
+                    'instruction' => 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                                      incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+                                      exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
+                                      irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat
+                                      nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa
+                                      qui officia deserunt mollit anim id est laborum.'
+                ]
+            ],
+            'expectedDonorText' => ['You must have signed and dated continuation sheet/s 2 before you signed ' .
+                                    'section 9 of the LPA, or on the same day.'],
+            'expectedAttorneyText' => []
+        ],
+        // Combined CS1 and CS2
+        // 1. LPA has more than 4 people to notify (generates CS1)
+        // 2. additional information on when replacement attorneys should act (generates CS2)
+        [
+            'lpa' => [
+                'document' => [
+                    'peopleToNotify' => [[], [], [], [], []],
```

Comment:
worth a comment about the empty arrays here - that we don't care about any of the people's details, just that they are present